### PR TITLE
fix(frontend): Support of suspend & stop expression + restore import/export menu

### PR DESCRIPTION
**Pull request by fatonramadani**
- Support of suspend & stop expression 
- Restore import/export menu
- Change split panel separator color

### CODE REVIEW

File: frontend/src/lib/components/flows/content/FlowModule.svelte [hunk 1]
```diff
@@ -161,6 +172,20 @@
...
+											{#if ('path' in flowModule.value && flowModule.value.path) || ('language' in flowModule.value && flowModule.value.language)}
+												<input
+													on:click|stopPropagation={() => undefined}
+													class="overflow-x-auto"
+													type="text"
+													bind:value={flowModule.summary}
+													placeholder="Summary"
+												/>
+											{/if}
+										</TabContent>
```
Comment (rubenfiszel) [inline frontend/src/lib/components/flows/content/FlowModule.svelte]: let's remove, we can already set it in 2 places normally. In the header of the module and in the minimap

File: frontend/src/lib/components/flows/content/FlowModuleAdvancedSettings.svelte [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	import Toggle from '$lib/components/Toggle.svelte'
+
+	import type { FlowModule } from '$lib/gen'
+
+	export let flowModule: FlowModule
+</script>
+
+<div class="font-bold text-sm mb-2">Suspend</div>
+<div class="space-y-4 ">
+	<input bind:value={flowModule.suspend} type="number" min="0" placeholder="0" />
```
Comment (rubenfiszel) [inline frontend/src/lib/components/flows/content/FlowModuleAdvancedSettings.svelte]: Suspend may also be undefined. Although it has the same effect as suspend = 0, I think it is cleaner to do the same logic as below with enable/disable, except the new min would be 1

File: frontend/src/lib/components/flows/content/FlowModule.svelte (hunk 1 above)
Comment (fatonramadani) [reply frontend/src/lib/components/flows/content/FlowModule.svelte]: Done

File: frontend/src/lib/components/flows/content/FlowModuleAdvancedSettings.svelte (hunk 2 above)
Comment (fatonramadani) [reply frontend/src/lib/components/flows/content/FlowModuleAdvancedSettings.svelte]: Done

**Pull request merged**